caikit/interfaces/runtime/data_model/training_management.py

```diff
@@ -18,37 +18,28 @@
 
 # First Party
 from py_to_proto.dataclass_to_proto import Annotated, FieldNumber
-import alog
 
 # Local
-from caikit.core.data_model import DataObjectBase, TrainingStatus, dataobject
-from caikit.core.toolkit.wip_decorator import Action, WipCategory, work_in_progress
+from ....core.data_model import DataObjectBase, TrainingStatus, dataobject
+from .package import RUNTIME_PACKAGE
 
-log = alog.use_channel("MDLOPS")
 
-RUNTIME_PACKAGE = "caikit_data_model.runtime"
-
-
-@work_in_progress(action=Action.WARNING, category=WipCategory.BETA)
```

Comment:
This isn't really doing anything at this point beyond spitting out warnings. It's in real usage, so it's no longer WIP.